content/sensu-core/0.29/platforms/sensu-on-rhel-centos.md

```diff
@@ -15,26 +15,27 @@ menu:
 
 ## Reference documentation
 
-- [Installing Sensu Core](#sensu-core)
-  - [Install Sensu using YUM](#install-sensu-core-repository)
-- [Installing Sensu Enterprise](#sensu-enterprise)
-  - [Install the Sensu Enterprise repository](#install-sensu-enterprise-repository)
-  - [Install Sensu Enterprise (server & API)](#install-sensu-enterprise)
-- [Configure Sensu](#configure-sensu)
-  - [Create the Sensu configuration directory](#create-the-sensu-configuration-directory)
-  - [Example client configuration](#example-client-configuration)
-  - [Example transport configuration](#example-transport-configuration)
-  - [Example data store configuration](#example-data-store-configuration)
-  - [Example API configurations](#example-api-configurations)
-    - [Standalone configuration](#api-standalone-configuration)
-    - [Distributed configuration](#api-distributed-configuration)
-  - [Example Sensu Enterprise Dashboard configurations](#example-sensu-enterprise-dashboard-configurations)
-    - [Standalone configuration](#dashboard-standalone-configuration)
-    - [Distributed configuration](#dashboard-distributed-configuration)
-  - [Enable the Sensu services to start on boot](#enable-the-sensu-services-to-start-on-boot)
-  - [Disable the Sensu services on boot](#disable-the-sensu-services-on-boot)
-- [Operating Sensu](#operating-sensu)
-  - [Managing the Sensu services/processes](#service-management)
+- [Sensu on RHEL/CentOS](#sensu-on-rhelcentos)
```

Comment:
A bunch of these ToC changes have left broken links. Should this sort of change be broken out into another issue possible? I notice other platforms still follow the old ToC setup, so may be worth a new issue to get all of them at the same time.